src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public abstract double getArea();
```

Comment:
better move method getArea() to separate interface, for example, AreaCalculation